src/components/Avatar/Avatar.stories.tsx

```diff
@@ -60,13 +60,13 @@ export const WithUsername: Story = {
   render: (args) => (
     <HStack spacing={16}>
       <VStack>
-        {["md", "sm"].map((size) => (
-          <Avatar size={size} {...args} />
+        {["md", "sm"].map((size, idx) => (
+          <Avatar key={idx} size={size} {...args} />
```

Comment:
Personally try to avoid using `key={idx}` wherever possible if we have another unique variable to assign instead. In this case it's unlikely to cause any issues, but wanted to raise to see if folks care to standardize our approach with these.

Reply:
I agree here for consistency of using unique ids or similar where possible.